plugins/inputs/aos/aos.go

```diff
@@ -304,6 +350,26 @@ func (ssl *StreamAos) ExtractFileInfo(fileInfo []*aos_streaming.FileInfo, origin
 	}
 }
 
+func (ssl *StreamAos) reportMessageLoss(msg_type string, expected uint64, actual uint64) {
+	ssl.msgLoss.Lock()
+	defer ssl.msgLoss.Unlock()
+	if actual < expected {
+		log.Printf("W! %v sequence number less than expected sequence number.", msg_type)
+		return
+	}
+
+	loss_count := actual - expected
+	serie := "message_loss"
```

Comment:
series would be better name  right?

Reply:
Yes - I considered that. This is an artifact from the original code - not sure why this name was chosen. I'm wondering if it might be better to call it metric.